templates/components/button.html.twig

```diff
@@ -22,9 +20,10 @@
         {% set attr = values.attr ?? {} %}
         {% set forceTitle = values.combined ?? false %}
         {% set label = values.label ?? null %}
```

Comment:
I don't really like `label` as badge data.
I was really lost when I saw it (without doc), I though it was the label for the button.

IMO it needs to be called `badge` and `badgeColor`

Reply:
i am using the entire macro in my application in hundreds of places. you can change it to
`{% set badge = values.label ?? (values.badge ?? null) %}` or even the other way around, but we need to keep the support for label

Reply:
Yeah we can define it like that.
But in the other way, to recommend to devs to use badge instead.

`{% set badge = values.badge ?? (values.label ?? null) %}`